sycl/source/CMakeLists.txt

```diff
@@ -92,6 +92,7 @@ function(add_sycl_rt_library LIB_NAME LIB_OBJ_NAME)
   else()
     target_compile_options(${LIB_OBJ_NAME} PUBLIC
                            -fvisibility=hidden -fvisibility-inlines-hidden)
+    target_compile_options(${LIB_OBJ_NAME} PUBLIC -Wno-psabi)
```

Comment:
In-sources comment with a motivation/description would be helpful here, I think

Reply:
done